pyRF24/pyRF24.cpp

```diff
@@ -285,8 +285,8 @@ BOOST_PYTHON_MODULE(RF24)
     // ******************** RF24 class  **************************
     bp::class_<RF24>("RF24", bp::init<uint16_t, uint16_t>((bp::arg("_cepin"), bp::arg("_cspin"))))
 #if defined(RF24_LINUX) && !defined(MRAA)
-        .def(bp::init<uint16_t, uint16_t, uint32_t>((bp::arg("_cepin"), bp::arg("_cspin"), bp::arg("spispeed"))))
-        .def(bp::init<uint32_t>((bp::arg("spispeed"))))
+        .def(bp::init<uint16_t, uint16_t, uint32_t>((bp::arg("_cepin"), bp::arg("_cspin"), bp::arg("spi_speed"))))
+        .def(bp::init<uint32_t>((bp::arg("spi_speed"))))
```

Comment:
This is the only questionable change because it could break existing user code that invokes the optional SPI speed parameter as a keyword argument. The following patch would need to be applied if this is merged.
```diff
-radio = RF24(CE_PIN, CSN_PIN, spispeed=4000000)
+radio = RF24(CE_PIN, CSN_PIN, spi_speed=4000000)
```
although any code that invokes it as a positional argument should still work fine
```python
radio = RF24(CE_PIN, CSN_PIN, 4000000)
```
I'm betting that most users are doing the second usage (if at all changing the default SPI speed) because the C++ docs (including COMMON_ISSUES.md) suggest doing it this way.

Reply:
IIRC, boost.python does _not_ handle keyword arguments gracefully. It may be safe to merge this if `spispeed=<int>` never worked previously.